mackup/application.py

```diff
@@ -45,18 +61,21 @@ def backup(self):
 
         # For each file used by the application
         for filename in self.files:
-            # Get the full path of each file
-            filepath = os.path.join(os.environ['HOME'], filename)
-            mackup_filepath = os.path.join(self.mackup.mackup_folder, filename)
+            (home_filepath, mackup_filepath) = self.getFilepaths(filename)
 
             # If the file exists and is not already a link pointing to Mackup
-            if ((os.path.isfile(filepath) or os.path.isdir(filepath))
-                and not (os.path.islink(filepath)
+            if ((os.path.isfile(home_filepath) or os.path.isdir(home_filepath))
+                and not (os.path.islink(home_filepath)
                          and (os.path.isfile(mackup_filepath)
                               or os.path.isdir(mackup_filepath))
-                         and os.path.samefile(filepath, mackup_filepath))):
+                         and os.path.samefile(home_filepath, mackup_filepath))):
+
+                print("Backing up {} ...".format("" if self.verbose else filename))
```

Comment:
Can you make an explicit if statement, I like to keep those construction to really dumb statements.